packages/calcite-components/src/utils/globalScript.ts

```diff
@@ -1,20 +1,18 @@
+/**
+ * This file is imported in src/runtime.ts
+ */
 import { isServer } from "lit";
 import { initModeChangeEvent } from "./mode";
 import { stampVersion } from "./config";
```

Comment:
I found it a bit curious that stampVersion will not log the 2nd time if multiple calcite versions are present - dual-bundling can lead to weird errors happening, so it is good to make developer aware of it early.

but, I was able to abuse the fact that sampVersion won't wire the 2nd time by adding `(globalThis as { calciteConfig?: { version?: string } }).calciteConfig ??= { version: " " };` to silence this message in Lumina dev server
(since Lumina controls importing calcite, detecting dual bundling is less important)

Reply:
Good observation. It used to warn when handling a previously stamped version, but it was causing confusion (see https://github.com/Esri/calcite-design-system/issues/9721#issuecomment-2229490202 and https://github.com/Esri/calcite-design-system/pull/9739).